pkg/noop/service.go

```diff
@@ -45,3 +45,7 @@ func (s *ServiceUnknown) PostAPICall(result interface{}) error {
 func (s *ServiceUnknown) Name() string {
 	return "ServiceUnknown"
 }
+
+func (s *ServiceUnknown) ExecuteAsLastStep(ctx configuration.Context) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ServiceUnknown.ExecuteAsLastStep should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)